app/javascript/stylesheets/theme.scss

```diff
@@ -2,12 +2,26 @@ html, body {
   height: 100%;
 }
 
-pre code.hljs {
-  padding: 20px;
+pre {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
I don't actually see this being used anywhere. Any reason you can't just use `m-0 p-0`?